plugin.py

```diff
@@ -90,6 +93,18 @@ def install_or_update(cls) -> None:
             raise
 
 
+class RustAnalyzerOpenDocsCommand(LspTextCommand):
+    session_name = "rust-analyzer"
+
+    def run(self, edit: sublime.Edit) -> None:
+        params = text_document_position_params(self.view, self.view.sel()[0].b)
+        session = self.session_by_name(self.session_name)
+        session.send_request(Request('experimental/externalDocs', params), self.on_result)
+
+    def on_result(self, url: Optional[str]) -> None:
+        if url is not None:
+            self.view.window().run_command("open_url", { "url": url })
```

Comment:
While not likely to be a big issue, various things should be "null-checked" for the code to be really fool proof.

 - The `sel()` can be empty
 - The `session` can be `None`
 - The `window()` can be `None`